controllers/managedocs_controller.go

```diff
@@ -925,26 +1008,28 @@ func (r *ManagedOCSReconciler) reconcilePrometheusService() error { //nolint:dea
 
 		r.prometheusService.Spec.Ports = []corev1.ServicePort{
 			{
-				Name:       "odf-prometheus-endpoint",
+				Name:       "https",
 				Protocol:   corev1.ProtocolTCP,
 				Port:       9339,
-				TargetPort: intstr.FromString("web"),
+				TargetPort: intstr.FromString("https"),
 			},
 		}
 		r.prometheusService.Spec.Selector = map[string]string{
-			monLabelKey: monLabelValue,
+			prometheusServiceSelectorLabelKey: prometheusServiceSelectorLabelValue,
 		}
 		utils.AddAnnotation(r.prometheusService,
 			servingCertSecretBetaAnnotationKey, prometheusServiceSecretName)
 		utils.AddAnnotation(r.prometheusService,
 			servingCertSecretAlphaAnnotationKey, prometheusServiceSecretName)
+		utils.AddLabel(r.prometheusService,
+			prometheusServiceLabelKey, prometheusServiceLabelValue)
 		return nil
 	})
 	return err
 }
 
-func (r *ManagedOCSReconciler) reconcileServingCertCABundleConfigMap() error { //nolint:deadcode,unused
-	r.Log.Info("Reconciling servingCertCABundleConfigMap")
+func (r *ManagedOCSReconciler) reconcileServingCertCABundleConfigMap() error {
```

Comment:
The annotation names in this function should be inlined as of the reasoning in the comment above about the service serving cert annotations 

Also, line splitting styling.

Reply:
Done in latest commit